internal/backend/svc/banner.go

```diff
@@ -27,7 +27,7 @@ var banner string
 
 var bannerTemplate = template.Must(template.New("banner").Parse(banner))
 
-func printBanner(cfg *bannerConfig, opts RunOptions, addr, wpAddr, temporalFrontendAddr, temporalUIAddr string) {
+func printBanner(cfg *bannerConfig, opts RunOptions, addr, wpAddr, wpVersion, temporalFrontendAddr, temporalUIAddr string) {
```

Comment:
You've officially crossed the threshold of "too many parameters, better use a struct".

Also, "addr" and "wp*" are unclear param names.

Also, please document this function, and mention the boolean config key to control showing this banner (`banner.show`).

I'm ok with doing this in a separate PR, but if so please do it soon-ish.

Reply:
in a futute pr.